maintainer-nomination-docs/discord-message-template.md

```diff
@@ -31,12 +31,14 @@ Use this space to write a personal message in support of the nominee becoming ma
 ### Your Responsibility as a Voter:
 *(Please include this section as written, in its entirety)*
 
-Please reply to this thread with a clear vote within three days: yes, no, or abstain. Not voting within three days is equivalent to abstaining.
+Please vote in the poll on or before [3 business days in the future]. Not voting within this period is equivalent to abstaining.
```

Comment:
Is this going to be a “no-one remembers to fill in the blank”, Sarah? 😂

Reply:
Haha, now I see @HiDeoo’s review, I’m even less sure this format is a good idea 😁

Reply:
s tahahah... alright, agreed! Let's figure out what should go here!

Reply:
Oh, just realized what this is thanks to your review 🤣 Maybe the "(Please include this section as written, in its entirety)" should be edited then?